packages/admin/src/table/filterbar/FilterBarField.tsx

```diff
@@ -0,0 +1,218 @@
+import { Button, ButtonProps, Popover, Typography } from "@material-ui/core";
+import { makeStyles, Theme, ThemeOptions, useTheme } from "@material-ui/core/styles";
+import { FieldState } from "final-form";
+import { isEmpty, isEqual } from "lodash";
```

Comment:
"lodash" is not in package.json, only "lodash.xxx" is, so we should use those for imports